code/modules/lighting/lighting_source.dm

```diff
@@ -358,7 +358,7 @@
 			if ((DETERMINANT(limit_a_x, limit_a_y, test_x, test_y) > 0) || DETERMINANT(test_x, test_y, limit_b_x, limit_b_y) > 0)
 				continue
 
-		if (TURF_IS_DYNAMICALLY_LIT_UNSAFE(T) || T.light_source_solo || T.light_source_multi)
+		if (TURF_IS_DYNAMICALLY_LIT_UNSAFE(T) || T.corners || T.light_source_solo || T.light_source_multi)
```

Comment:
I'm pretty sure the correct fix is:
```suggestion
		if (TURF_IS_DYNAMICALLY_LIT_UNSAFE(T) || T.ambient_light || T.light_source_solo || T.light_source_multi)
```

Reply:
corners is wrong, but ambient_light doesn't cover the case where space ambience/starlight is disabled.